presentation/common/src/presentation-common/hierarchy/Key.ts

```diff
@@ -274,3 +278,51 @@ export interface LabelGroupingNodeKeyJSON extends GroupingNodeKeyJSON {
  * @public
  */
 export type NodeKeyJSON = BaseNodeKeyJSON | ECInstancesNodeKeyJSON | ECClassGroupingNodeKeyJSON | ECPropertyGroupingNodeKeyJSON | LabelGroupingNodeKeyJSON;
+
+/**
+ * Data structure that describes a presentation query
+ * @alpha
+ */
+export interface PresentationQuery {
+  /** ECSQL query */
+  query: string;
+  /** The query bindings */
+  bindings?: PresentationQueryBinding[];
+}
+
+/** @alpha */
+export interface BasePresentationQueryBinding {
+  type: "Id" | "IdSet" | "ECValue" | "ValueSet";
+}
+
+/** @alpha */
+export interface IdBinding extends BasePresentationQueryBinding {
+  type: "Id";
+  value: Id64String;
+}
+
+/** @alpha */
+export interface IdSetBinding extends BasePresentationQueryBinding {
+  type: "IdSet";
+  value: Id64String[];
+}
+
+/** @alpha */
+export interface ECValueBinding extends BasePresentationQueryBinding {
+  type: "ECValue";
+  valueType: string;
+  value: any;
+}
+
+/** @alpha */
+export interface ECValueSetBinding extends BasePresentationQueryBinding {
+  type: "ValueSet";
+  valueType: string;
+  value: any[];
+}
+
+/**
+ * One of the presentation query binding types
+ * @public
+ */
+export type PresentationQueryBinding = IdBinding | IdSetBinding | ECValueBinding | ECValueSetBinding;
```

Comment:
Why is this `public` if all types used in union are `alpha`? I think either they all should be `public` or this one should be `alpha`.

Reply:
Made `alpha`.